engine/evarutil.mli

```diff
@@ -136,6 +136,7 @@ val tj_nf_evar :
 
 val nf_named_context_evar : evar_map -> Constr.named_context -> Constr.named_context
 val nf_rel_context_evar : evar_map -> rel_context -> rel_context
+val nf_rel_context_evar_with_relevance : evar_map -> rel_context -> rel_context
```

Comment:
Do we actually ever want the one that doesn't normalize relevance? It seems like we should modify nf_rel_context_evar instead of adding a new API.

Reply:
Please do it

Reply:
OK, done. And done also for named_context.